javascript/javascript2/week1/medianAndAverage.js

```diff
@@ -0,0 +1,53 @@
+function getMedian(values)//here median function
+{
+	values.sort(function(a,b){
+  	return a-b;
+  });
+  var half = Math.floor(values.length / 2);//median logics
+  let median;
+  if (values.length % 2)
+  {
+      median = values[half];
+  }
+  else
+  {
+      median = (values[half - 1] + values[half]) / 2.0;
+  }
+  return median;//returning median
+}
+
+function getAverage(arr) //function average
+{
+    var sum=0;
+    for(var i in arr)
+    {
+        sum = sum + arr[i];//logic
+    }
+    var average=sum/arr.length;//formulae
+    return average;//returning avg
+}
+
+const housePrices = [4000000, 6500000, 1400000, 70000000, 300000000, 9000000, 4400000];
+function medianAverage(arrayOfNumbers)//function,price and values assigning
+{
+    let averageVal = getAverage(housePrices);
```

Comment:
You shouldn't use let unless you are planning on re-assigning the variable with the `=` operator